WordPress/Docs/WhiteSpace/DisallowInlineTabsStandard.xml

```diff
@@ -0,0 +1,25 @@
+<documentation title="Disallow Inline Tabs">
+    <standard>
+    <![CDATA[
+    Use spaces for inline alignments.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: spaces used for inline alignment.">
+        <![CDATA[
+$a = array(
+    'a'<em>[space]</em>=> 'lor',
```

Comment:
May I suggest making this array key three characters long ?
That way the double arrows with the spacing used would be aligned (in real code), while with the current code example people may wonder why the alignment on the second line is done in the first place and why it is off by two.

```suggestion
    'abc'<em>[space]</em>=> 'lor',
```

Reply:
Sorry to be a pain, but this has now been fixed solely for the "Invalid" example, not (yet) for the "Valid" one :pensive:

Reply:
sorry! my bad :/
I pushed an update